app/models/snpedia_paper.rb

```diff
@@ -9,4 +9,13 @@ class SnpediaPaper < ActiveRecord::Base
   def summary
     read_attribute(:summary).presence || "No summary provided."
   end
+
+  def matches_local_genotype?(other_local_genotype)
+    local_genotype == other_local_genotype.split('').sort.join
+  end
+
+  def local_genotype
+    url =~ /\((.*);(.*)\)$/
+    $1 + $2
+  end
```

Comment:
SNPedia links are displayed bold if the nucleotide pair matches a SNP in your genotype. I moved this logic here. It is used [here](https://github.com/gedankenstuecke/snpr/pull/182/files#diff-8c0ff2a01904903c17a5dd6e62a2cc1eR3).